### DISCUSSION TITLE
Support existing database schema

### DISCUSSION THREAD

**Discussion by joelbqz** [keystonejs/keystone#5567] [Questions] [answered]
Hello guys! I love what you've done with Keystone.

I'm using Hasura(which works with Postgres) in a large web application project, is it possible to map keystone schema fields to an existing Postgres Database, so I don't have to replace my entire DB Schema.

Best!!

**Comment by bladey** [comment, marked as answer]
Hi @joelbqz, you should be able to use the existing database, as long as it is compatible with Keystones fields types, please join us on https://community.keystonejs.com if you have further questions!
